[WIP] Add Short Description Data

SofieVerrewaere opened the pull request:
ToDo:

- [x] Add a short description of the data in the README.md directly.
- [ ] Merge again with the main branch after `nbdev` is merged.

Transported changes to `index.ipynb` and generate `README.md` via `nbdev_build_docs`.

simeneide reviewed (changes requested):
Great effort! I'm on mobile and can't review the text but i have a bad piece of news: since we are using the nbdev framework you have to change index.ipynb instead of readme.md. then you run nbdev_build_docs (command line) and the framework generates readme from it.

SofieVerrewaere commented:
Just for documentation:

Errors when running `nbdev_build_docs`:
- first error with gem:
```
ERROR: While executing gem ... (Gem::FilePermissionError)
You don't have write permissions for the /Library/Ruby/Gems/2.6.0 directory
```
fixed with this [solution](https://www.moncefbelyamani.com/the-definitive-guide-to-installing-ruby-gems-on-a-mac/?utm_source=stackoverflow#install-ruby-with-a-manager) 

- second error with the conversion of `index.ipynb`:

```
'HTMLExporter' object has no attribute 'template_path'
```
fixed with `conda install nbconvert=5.6.1`
